tests.c

```diff
@@ -450,6 +450,35 @@ test_opcode_0xc6(void)
   cpu_write_mem(&cpu, 0x0001, 0x00);
 }
 
+void
+test_opcode_0xca(void)
+{ // JZ
+  i8080 cpu;
+  cpu_init(&cpu);
+
+  // case where zero flag not set
+  cpu.pc = 0x4341;                                  // NOLINT
+  cpu_write_mem(&cpu, cpu.pc + 1, 0x28);            // NOLINT
+  cpu_write_mem(&cpu, cpu.pc + 2, 0xb7);            // NOLINT
+  update_zero_flag(&cpu, 1);                        // had a remainder
+  int code_found = execute_instruction(&cpu, 0xca); // NOLINT
+  CU_ASSERT(code_found >= 0);
+  CU_ASSERT(cpu.pc == 0x4344); // NOLINT
+
+  // case where zero flag set
+  cpu.pc = 0x5441;                              // NOLINT
+  cpu_write_mem(&cpu, cpu.pc + 1, 0x3a);        // NOLINT
+  cpu_write_mem(&cpu, cpu.pc + 2, 0xa9);        // NOLINT
+  update_zero_flag(&cpu, 0);                    // no remainder
+  code_found = execute_instruction(&cpu, 0xca); // NOLINT
+  CU_ASSERT(code_found >= 0);
+  CU_ASSERT(cpu.pc == 0xa93a); // NOLINT
+
+  // cleanup
+  cpu_write_mem(&cpu, 0x0001, 0x00); // NOLINT
```

Comment:
What is this cleaning up? Shouldn't it clean up 0x4342, 0x4343, 0x5442, and 0x5443?

Reply:
Done & pushed